<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyConstraintTeacherNotAvailableTimesForm_template</class>
 <widget class="QDialog" name="ModifyConstraintTeacherNotAvailableTimesForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to modify a constraint of this type">Modify teacher not available times</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>This constraint does not induce gaps for teachers. If a teacher has lessons before and after this not available period, there will be no gaps counted</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="teacherGridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="teachersComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <property name="autoCompletion">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="teacherTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Teacher</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="notAllowedTimesTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="groupTextLabel">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="essentialRadioButton">
       <property name="text">
        <string>Essential</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="importantRadioButton">
       <property name="text">
        <string>Important</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="desirableRadioButton">
       <property name="text">
        <string>Desirable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="weightBoxLayout">
     <item>
      <widget class="QLabel" name="weightTextLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Weight percentage (necessary: 100%)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="buttonsGridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="setAllAvailablePushButton">
       <property name="text">
        <string comment="It refers to time slots">Set all available</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string extracomment="This is an explanation in a dialog for a constraint. It says that symbol X means that this slot is not allowed, and an empty cell means that the slot is allowed">X=not allowed, empty=allowed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="setAllNotAvailablePushButton">
       <property name="text">
        <string comment="It refers to time slots">Set all not available</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
